python/rpdk/python/codegen.py

```diff
@@ -233,6 +238,37 @@ def _make_pip_command(base_path):
             str(base_path / "build"),
         ]
 
+    @staticmethod
+    def _get_plugin_information(project):
+        requirements_file = project.root / "requirements.txt"
+        plugin_version = None
+
+        with open(requirements_file) as f:
+            line = f.readline()
+            while line:
+                line = line.strip()
+                if line.startswith(SUPPORT_LIB_NAME):
+                    line_without_prefix = line[len(SUPPORT_LIB_NAME) :]
+
+                    semi_colon = ";"
+                    if semi_colon in line_without_prefix:
+                        index = line_without_prefix.index(semi_colon)
+                        line_without_prefix = line_without_prefix[0:index].strip()
+
+                    plugin_version = re.split("=\\s*", line_without_prefix.strip())[-1]
+                    break
+
+                line = f.readline()
+
+        plugin_info = {
+            "plugin-tool-version": __version__,
+            "plugin-name": "python",
+        }
+        if plugin_version and plugin_version is not None:
+            plugin_info["plugin-version"] = plugin_version
+
+        return plugin_info
```

Comment:
I would not pick the version from requirements file, as we statically specify [`>=2.1.3`](https://github.com/aws-cloudformation/cloudformation-cli-python-plugin/blob/master/python/rpdk/python/templates/requirements.txt#L1)

I don't think lib version can be available during the boiler plate setup. The lib is pulled from PyPi after the repo is setup